velox/exec/tests/WindowTest.cpp

```diff
@@ -17,129 +17,23 @@
 #include "velox/common/file/FileSystems.h"
```

Comment:
Its a bit weird that this file is renamed as WindowTest and AggregateWindowTest is a new one. Shouldn't it be the other way around ? Please check if this can be fixed.

Reply:
Thanks for pointing it out, I found this to be strange too and am not sure why it happened. This incorrect renaming is reverted now, but `git mv SimpleAggregatesTest.cpp AggregateWindowTest.cpp` is not reflected in the commit. git adds the renamed file as a new file instead.

Reply:
This incorrect renaming appears to be a result of `make format-fix`, so I am not sure if this should be fixed. Could you please share how to proceed?